packages/@ama-terasu/core/project.json

```
@@ -14,13 +14,11 @@
       "dependsOn": ["compile"]
     },
     "compile": {
-      "executor": "@nx/js:tsc",
-      "outputs": ["{projectRoot}/dist", "{projectRoot}/build/.tsbuildinfo"],
+      "executor": "nx:run-script",
       "options": {
-        "main": "packages/@ama-terasu/core/src/public_api.ts",
-        "tsConfig": "packages/@ama-terasu/core/tsconfig.build.json",
-        "outputPath": "packages/@ama-terasu/core/dist"
-      }
+        "script": "build:source"
+      },
+      "outputs": ["{projectRoot}/dist", "{projectRoot}/build/.tsbuildinfo"]
```

Seems to be the `compile` task default output from `nx.json`
```suggestion
      }
```